voice-queue/tests/enqueue.protected.test.js

```diff
@@ -0,0 +1,25 @@
+const helpers = require('../../test/test-helper');
+const enqueue = require('../functions/enqueue.protected').handler;
+
+const context = {};
+
+const event = {
+  CallSid: 'CAX',
```

Comment:
We're missing a `From` here, so the function never tests whether a call is placed. Check out other functions, [like the airtable one](https://github.com/twilio-labs/function-templates/blob/main/airtable/test/broadcast-sms.test.js#L41-L53), to see how to mock `getTwilioClient` on the `context`.